core/src/com/fallenflame/game/PlayerModel.java

```diff
@@ -231,9 +236,10 @@ protected void setFireBuddyOrigin(float x, float y){
     /**
      * Get the position of the firebuddy on the screen in meters
      */
-    public Vector2 getFireBuddyPosition() { return new
-            Vector2(getPosition().x + (getFireBuddyOriginX() + fireBuddyFilmstrip.getRegionWidth()) / drawScale.x / 2.0f,
-            getPosition().y - (getFireBuddyOriginY() / drawScale.y) + fireBuddyFilmstrip.getRegionHeight() / drawScale.y / 2.0f);}
+    public Vector2 getFireBuddyPosition() { return new Vector2(
```

Comment:
Why is this changed here?

Reply:
I tried to factor things out into variables but then realized it would look even worse than it does now. There's no differenec in the code except I just changed the line of the initializer